components/ButtonGenerator/CodeBlock.tsx

```diff
@@ -70,6 +70,51 @@ export default function CodeBlock({ button }): JSX.Element {
     }
   };
 
+  const checkCurrency = (value, type) => {
+    if (value === 'XEC' || value === 'BCH') {
+      return '';
+    } else if (type === 'html') {
+      return `\n  currency="${value}"`;
+    } else if (type === 'js') {
+      return `\n    currency: "${value}",`;
+    } else {
+      return `\n  const currency = '${value}'`;
+    }
+  };
+
+  const primaryXEC = '#0074C2';
+  const secondaryXEC = '#FFFFFF';
+  const tertiaryXEC = '#231f20';
+
+  const primaryBCH = '#4bc846';
+  const secondaryBCH = '#FFFFFF';
+  const tertiaryBCH = '#231f20';
+
+  const checkTheme = (value, type) => {
+    if (
+      (value.palette.primary === primaryXEC &&
+        value.palette.secondary === secondaryXEC &&
+        value.palette.tertiary === tertiaryXEC) ||
+      (value.palette.primary === primaryBCH &&
+        value.palette.secondary === secondaryBCH &&
+        value.palette.tertiary === tertiaryBCH)
+    ) {
```

Comment:
How about re-writing this to support N number of currencies?

These colours should be in a const or conf file, ideally.